src/main/java/com/ffxivcensus/gatherer/player/PlayerBuilder.java

```diff
@@ -103,17 +104,18 @@ public static void setLevels(final PlayerBean player, final int[] arrLevels) {
         player.setLevelThaumaturge(arrLevels[12]);
         player.setLevelArcanist(arrLevels[13]);
         player.setLevelRedmage(arrLevels[14]);
-        player.setLevelCarpenter(arrLevels[15]);
-        player.setLevelBlacksmith(arrLevels[16]);
-        player.setLevelArmorer(arrLevels[17]);
-        player.setLevelGoldsmith(arrLevels[18]);
-        player.setLevelLeatherworker(arrLevels[19]);
-        player.setLevelWeaver(arrLevels[20]);
-        player.setLevelAlchemist(arrLevels[21]);
-        player.setLevelCulinarian(arrLevels[22]);
-        player.setLevelMiner(arrLevels[23]);
-        player.setLevelBotanist(arrLevels[24]);
-        player.setLevelFisher(arrLevels[25]);
+        player.setLevelBluemage(arrLevels[15]);
```

Comment:
I can't check right now, but *think* this would break the current gathering until the lodestone is updated, as there's nothing changed in how we're populating the `arrLevels` variable.

Probably worth leaving this change out, for the time being, as we can preempt the DB change and Model change, but nothing related to the lodestone.